lib/web/scripts/oneoff/oneoff.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
```

Comment:
As discussed, instead of generic "one-off" script, let's build a specific script that calls "teleport iam configure" command.

Reply:
The goal here is to create the generic script but it's not actually used anywhere
The next PR will add an endpoint that generates a oneoff script specific to the `teleport iam configure` command